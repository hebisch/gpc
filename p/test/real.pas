program writer(output);
begin
 writeln(Sin(1/2):0:10);
 writeln(SqRt(1/2):0:10);
 writeln(Ln(1/2):0:10);
 writeln(Exp(1/2):0:10);
 writeln(Cos(1/2):0:10);
end.
