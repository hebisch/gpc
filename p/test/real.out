0.4794255386
0.7071067812
-0.6931471806
1.6487212707
0.8775825619
